src/main/java/duke/storage/Deadline.java

```diff
@@ -0,0 +1,111 @@
+package duke.storage;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Deadline class defines a 'Deadline' task used for the application
+ *
+ * @author Ryan NgWH
+ */
+public class Deadline extends Task {
+    /**
+     * Due date/time of the deadline task
+     */
+    private Instant dueDate;
+
+    /**
+     * Create a Deadline task
+     *
+     * @param description Description of the deadline
+     * @param dueDate     Due date of the deadline
+     */
+    public Deadline(String description, Instant dueDate) {
+        super(description, TaskType.DEADLINE, false);
+        this.dueDate = dueDate;
+    }
+
+    /**
+     * Create a Deadline task
+     *
+     * @param description Description of the deadline
+     * @param dueDate     Due date of the deadline
+     * @param isDone      Status of the deadline
+     */
+    public Deadline(String description, Instant dueDate, boolean isDone) {
```

Comment:
Noticed that isDone and isCompleted means refers to the status of a task. Would it be better to uniformly name these two Boolean variables?

Reply:
Good catch! Will make the relevant changes